gpu_bdb/bdb_tools/text.py

```diff
@@ -16,24 +16,36 @@
 
 import os
 
-
 EOL_CHAR = "è"
 
 
 def create_sentences_from_reviews(
-    df, review_column="pr_review_content", end_of_line_char=EOL_CHAR
+    df, review_column="pr_review_content", end_of_line_char=EOL_CHAR,
 ):
+    import pandas as pd
     import cudf
-
-    sentences = df[review_column].str.tokenize(delimiter=end_of_line_char)
-
-    # expand the reviews
-    tk_cnts = df[review_column].str.token_count(delimiter=end_of_line_char)
-
+    import numpy as np
+    
+    if isinstance(df, cudf.DataFrame):
+        sentences = df[review_column].str.tokenize(delimiter=end_of_line_char)   
+        tk_cnts = df[review_column].str.token_count(delimiter=end_of_line_char)
+    else:
+        sentences = df[review_column].str.split(end_of_line_char)
+        tk_cnts = sentences.str.len()
+        sentences = sentences.explode(ignore_index=True)
```

Comment:
I think we should stick to the same API as we plan to do benchmarking and not special case `cuDF`.  

```suggestion
    sentences = df[review_column].str.split(end_of_line_char)
    tk_cnts = sentences.str.len()
    sentences = sentences.explode(ignore_index=True)
```